test/vcf/assembly_checker_integration_test.cpp

```diff
@@ -182,6 +182,17 @@ namespace ebi
             CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, vcf_path, fasta, assembly_report, outputs));
         }
 
+        SECTION("with correct mappoing, no fasta provided, using ENA API to retrieve fasta for contigs, but got no sequence, fail")
```

Comment:
I'm not sure I understand what this means: "with correct mapping but got no sequence".

Reply:
I have reworded the description in 0edf59b. Hope it is now clearer.